experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -72,6 +78,10 @@ function Base.show(io::IO, R::RootSystem)
   end
 end
 
+@attr ZZMatrix function cartan_bilinear_form(R::RootSystem)
```

Comment:
I think we agreed that here `cartan` is not necessary.

```suggestion
@attr ZZMatrix function bilinear_form(R::RootSystem)
```